### DISCUSSION TITLE
No output web page

### DISCUSSION THREAD

**Discussion by ElleWilte** [community/community#32042] [Other Feature Feedback, Questions, & Ideas]
The link 'https://ellewilte.github.io/huiswerkopgave_les9/FortRammekens.html' generates the correct output. However, the line '<iframe src=["http://www.fortrammekens.nl/"](http://www.fortrammekens.nl/) width="560" height="700"></iframe>' in the HTML file gives an empty frame. Without GitHub, the external page of 'http://www.fortrammekens.nl/' is displayed in the output. Please your help. Thanks in advance.

**Comment by airtower-luna** [comment]
You're trying to embed an insecure (plaintext HTTP) document in a secure (HTTPS) web site. Most browsers block that for security. I'd recommend you change the URL to HTTPS, but that website doesn't seem to have a working HTTPS version. :confused:
